src/createAnimatedComponent.js

```diff
@@ -72,12 +72,14 @@ function createAnimatedComponent(Component: any): any {
     }
 
     render() {
-      const { style, ...other} = this._propsAnimated.__getValue();
+      const props = this._propsAnimated.__getValue();
+      if (props.style) {
+        props.style = ApplyAnimatedValues.transformStyles(props.style)
```

Comment:
This change mutates the object, previously it did not.

Reply:
Yes I realize that, but attempting to avoid an if statement with rendering the component and or creating another new object to spread on w/ props. 
The `__getValue` always returns a new object https://github.com/animatedjs/animated/blob/master/src/AnimatedProps.js#L37

Reply:
Gotcha. Why avoid an if statement?

Reply:
I don't know, personal preference.
Generally speaking I like my render functions to have a single return.

```
render() {
      const props = this._propsAnimated.__getValue();
      if (props.style) {
        return (
          <Component
            {...props}
            style={ApplyAnimatedValues.transformStyles(props.style)}
            ref={refName}
          />
        );
      }

      return <Component {...props} ref={refName} />;
    }
```

This is fine by me, accomplishes the same thing.